### ISSUE TITLE
Issues with stable version

### ISSUE THREAD

**Issue by enoch85** [nextcloud/vm#6]
- [X] Install script breaks here: https://github.com/nextcloud/vm/blob/master/nextcloud_install_testing.sh#L461 with `set -e` activated.

- [X] index.php is not included

- [X] nextcloud-startup-script.sh is not included

- [x] Instruction Boxes are a bit off

- [x] index.php is not activated, check apache config

**Comment by ezraholm50** [comment]
I ran the test script until line 410 (webmin included) and it all went fine. Then I ran the second part from 460 with the VARs and it gave me:

```
root@ubuntu:~# bash 2
sudo: unable to resolve host nextcloud
E: Command line option 'i' [from -ic] is not understood in combination with the other options.
```

After removing everything but the directory and repository VARs everything went fine, redis, php mods etc.
 
- [ ] ncadmin .profile isnt changed, startup info from initial start is still there and not the figlet
- [ ] Cant vind the trusted ip.sh script anywhere, it also doesnt work, nothing gets added to config.php
- [ ] Scripts doesnt ask to change passwords yet.
